framework/src/modules/chain/submodules/loader.js

```diff
@@ -867,7 +867,7 @@ __private.loadBlocksFromNetwork = function(cb) {
  */
 __private.sync = function(cb) {
 	library.logger.info('Starting sync');
-	if (components.cache) {
+	if (components.cache.cacheReady) {
```

Comment:
Why not we rename this attribute to just `ready`, it does not read well `cache.cacheReady`

Reply:
@nazarhussain I will create an issue to do that in the refactor epic (for which we included changes to the cache already)

Reply:
Created https://github.com/LiskHQ/lisk-sdk/issues/3606 . Resolving.